src/app/nasjonal/ferdigstilt/[sykmeldingId]/page.tsx

```diff
@@ -8,10 +8,10 @@ export const metadata: Metadata = {
     title: 'Rediger mangler i nasjonal sykmelding',
 }
 
-async function Page({ params }: { params: Promise<{ sykmeldingId: string }> }): Promise<ReactElement> {
-    const { sykmeldingId } = await params
+async function Page({ params }: { params: Promise<{ oppgaveId: string }> }): Promise<ReactElement> {
+    const { oppgaveId } = await params
```

Comment:
Params er fortsatt `sykmeldingId` her, ref. `nasjonal/ferdigstilt/[sykmeldingId]`. Bruker blir sendt til denne routen frå Modia med sykmeldingId. 

Ser også at graphql endepunktet for å hente ferdigstilt ikkje er oppretta i backend enda, så må nok vente med å gjere endringa for ferdigstilt til det er på plass i backend først.

Reply:
Bra du så.. vi må jo egentlig lage ferdig backenden først, eller ha to forskjellige view, siden NasjonalSykmeldingForm er brukt båd av NasjonalOppgaveView og NasjonalFerdigstiltView